Lib/asyncio/taskgroups.py

```diff
@@ -135,6 +150,9 @@ async def __aexit__(self, et, exc, tb):
                 self._errors = None
 
     def create_task(self, coro, *, name=None, context=None):
+        """Create a new task in this group and return it.
+        Similar to `asyncio.create_task`.
```

Comment:
Multi-line docstrings should have a blank line after the first line.
```suggestion
        """Create a new task in this group and return it.

        Similar to `asyncio.create_task`.
```